app/dashboard/views/alias_contact_manager.py

```diff
@@ -150,6 +151,65 @@ def get_contact_infos(
     return ret
 
 
+def create_contact(alias: Alias, contact_address: str) -> Optional[Contact]:
+    if not current_user.is_premium():
+        flash(f"Please upgrade to premium to create reverse-alias")
```

Comment:
I think `flash` requires a second parameter for the message level (info, warning, etc.)

Reply:
Oops. Good catch.